.circleci/config.yml

```diff
@@ -47,9 +47,6 @@ workflows:
           name: helm_lint
       - hmpps/build_docker:
           name: build_docker
-          snyk-fail-build: false
```

Comment:
Removing Snyk from docker build - it's not particularly helpful as at the moment it's incorrectly reporting vulnerabilities since it's unable to properly parse the dependency tree...